src/plugins/chat.ts

```
@@ -48,6 +54,807 @@ interface StreamedResponseContentChunk {
+
+        return result
+    }
+
+    const quote =
+        typeof args.quote === 'string' && args.quote.length > 0
+            ? ` quote="${escape(args.quote, true)}"`
+            : ''
+
+    if (Array.isArray(args.parts)) {
+        const content = args.parts
+            .filter((item) => item && typeof item === 'object' && !Array.isArray(item))
```

ESLint

[error] Replace `(item)·=>·item·&&·typeof·item·===·'object'·&&·!Array.isArray(item)` with `⏎················(item)·=>⏎····················item·&&·typeof·item·===·'object'·&&·!Array.isArray(item)⏎············`

(prettier/prettier)